plugins/idamagic/main_interface/__init__.py

```diff
@@ -84,6 +86,50 @@ def __init__(
         if not autoinst:
             self.parent.parent().parent().setSizes([1200, 1])
 
+    def check_ida_version(self):
+        """
+        Check if IDA version is 8.x.
+
+        Returns:
+            bool: True if version is 8.x, False otherwise
+        """
+        try:
+            # version = ida_kernwin.get_kernel_version()
+            version = "9.1"
```

Comment:
Is the hardcoded version incorrect here? I don't believe 9.1 exists yet?

Reply:
Another leftover, should not be here. I did this for testing and didn't `restore` correctly.